src/Controller/Component/AuthComponent.php

```diff
@@ -349,7 +349,7 @@ protected function _unauthenticated(Controller $controller)
         }
 
         if (!empty($this->_config['ajaxLogin'])) {
-            $controller->viewPath = 'Element';
+            $controller->getview()->viewPath = 'Element';
```

Comment:
Yo dawg, I heard you like clean code... So Suggested more cleanup inside your cleanup PR!